src/bindings/js/node/src/tensor.cpp

```diff
@@ -44,7 +44,8 @@ Napi::Function TensorWrap::get_class(Napi::Env env) {
                         InstanceMethod("getShape", &TensorWrap::get_shape),
                         InstanceMethod("getElementType", &TensorWrap::get_element_type),
                         InstanceMethod("getSize", &TensorWrap::get_size),
-                        InstanceMethod("isContinuous", &TensorWrap::is_continuous)});
+                        InstanceMethod("isContinuous", &TensorWrap::is_continuous), 
```

Comment:
As you can see in the [CI](https://github.com/openvinotoolkit/openvino/actions/runs/14088230946/job/39493821086?pr=29760), this space is unnecessary. Please fix other problems on your own.
```suggestion
                        InstanceMethod("isContinuous", &TensorWrap::is_continuous),
```